### PR TITLE
k830 support + support for NEW FN INVERSION

### PR DESCRIPTION
**Pull request by decibyte**

### CODE REVIEW

File: docs/devices.md [hunk 1]
```diff
@@ -151,6 +152,7 @@ Mouse-Keyboard combos:
 [K700]: http://logitech.com/product/wireless-desktop-mk710
 [K750]: http://logitech.com/product/k750-keyboard
 [K800]: http://logitech.com/product/wireless-illuminated-keyboard-k800
+[K830]: http://www.logitech.com/en-us/product/living-room-keyboard-k830
```
Comment (Lekensteyn) [inline docs/devices.md]: Just for standards sake, this URL also works: http://logitech.com/product/living-room-keyboard-k830

**Comment by Lekensteyn** [comment]
Looks good to me, can you squash the three documentation changes, and squash the two device support changes into one?

**Comment by decibyte** [comment]
Ahh! I didn't see the locale stuff in the URL. I'll fix that.

I can do the squashing, but is it a problem that changes are spread across several commits, as long as the end result works?

**Comment by decibyte** [comment]
Strictly speaking, there are 3 changes in this:
 * Support for NEW_FN_INVERSION
 * Support for k830.
 * Updated docs for k830.

Do you still want it to be squashed into 2 commits, or do you agree that 3 commits would be more correct?

**Comment by decibyte** [comment]
Gah... I somehow managed to close this PR. I'll open another.

**Pull request closed without merging**